docs/install_windows.rst

```diff
@@ -68,8 +68,31 @@ Manually installing dependencies
 Creating a Virtual Environment
 ------------------------------
 
+.. tip::
+
+    If you want to learn more about virtual environments, see page: `about-venvs`
+
 We require installing Red into a virtual environment. Don't be scared, it's very
-straightforward. See the section `installing-in-virtual-environment`.
+straightforward.
+
+First, choose a directory where you would like to create your virtual environment. It's a good idea
+to keep it in a location which is easy to type out the path to. From now, we'll call it
+``redenv`` and it will be located in your home directory.
+
+Start with opening a command prompt (open Start, search for "command prompt", then click it)
```

Comment:
Add a warning here and tell them to ensure they close power shell and open command prompt please